src/lao.c

```diff
@@ -323,6 +353,14 @@ static int l_is_big_endian(lua_State *L)
 	return 1;
 }
 
+static int l_append_global_option(lua_State *L)
+{
+	const char *key = luaL_checkstring(L, 1);
+	const char *val = luaL_checkstring(L, 2);
```

Comment:
OK - and `false` to "no", I guess ? Unusual, isn't it, that `nil` means `true` while `false` means `false`, but it comes logically from their config file.